api-report/test-runtime-utils.api.md

```diff
@@ -67,7 +67,9 @@ export interface IMockContainerRuntimePendingMessage {
 
 // @public
 export class InsecureTokenProvider implements ITokenProvider {
-    constructor(tenantKey: string, user: IUser);
+    constructor(
+    tenantKey: string,
+    user: IUser);
```

Comment:
Do you know how api-extractor formats its output? I've noticed these sorts of changes in other PRs, and I think they imply that we're not configuring api-extractor consistently in our configs.